includes/Product_Categories.php

```diff
@@ -47,7 +46,7 @@ public static function get_google_product_category_id( $category_id ) {
 	 */
 	public static function update_google_product_category_id( $id, $category_id ) {
 
-		// TODO: implement
+		add_term_meta( $id, Products::GOOGLE_PRODUCT_CATEGORY_META_KEY, $category_id, true );
```

Comment:
we need to use `update_term_meta()`, no? With `$unique = true` `add_term_meta()` bails if the meta key already exists so subsequent updates won't persist.